### DISCUSSION TITLE
Did Playwright get updated to 1.44 with the latest 2.0.12 of Artillery ?

### DISCUSSION THREAD

**Discussion by HugoImaios** [artilleryio/artillery#2749] [Artillery]
I'm getting the classic `Looks like Playwright Test or Playwright was just updated to 1.44.0.` when trying to run a test with the latest artillery release, but the Playwright version update is not mentionned in [the changelog](https://github.com/artilleryio/artillery/releases/tag/artillery-2.0.12).

**Comment by HugoImaios** [comment]
I just [compared](https://github.com/artilleryio/artillery/compare/artillery-2.0.11...artillery-2.0.12) 2.0.11 and 2.0.12 and saw this commit : https://github.com/artilleryio/artillery/commit/5374048ca35b86a15f6e99c1faa99ef0a211be15 so I guess it's just a lapse of memory :stuck_out_tongue:

**Reply by hassy** [reply]
Yes, we missed that in the changelog! sorry about that, will update it now.

**Reply by bernardobridge** [reply]
[Docs](https://www.artillery.io/docs/reference/engines/playwright#playwright-compatibility) are also updated now! Sorry for the lapse.

**Reply by HugoImaios** [reply]
no problem folks ! Thanks for the update, and the update of the doc (I was gonna add it to the description too haha)
